packages/web/src/scss/tools/_form-fields.scss

```
@@ -0,0 +1,136 @@
+@use 'sass:map';
+@use '../theme/form-fields' as form-fields-theme;
+@use '../settings/form-fields' as form-fields-settings;
+@use 'accessibility';
+@use 'typography';
+
+@mixin root($fieldType) {
+    @if $fieldType == 'inline' {
+        display: inline-flex;
+        margin-block: form-fields-theme.$input-spacing;
+    }
+
+    @if $fieldType == 'box' {
+        display: inline-block;
+        width: form-fields-theme.$box-field-input-width;
+    }
+}
+
+@mixin label($fieldType) {
```

```suggestion
@mixin label($field-type) {
```

Please check for other possible occurences.